kieker-tools/build.gradle

```diff
@@ -21,6 +21,10 @@ dependencies {
 	compile 'org.apache.commons:commons-math3:3.6.1'	// used by OPAD
 	compile 'org.apache.commons:commons-lang3:3.6'		// used by OPAD
 	compile configurations.resultJar
+	
+	compile group: 'log4j', name: 'log4j', version: '1.2.17'
+	compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
```

Comment:
As SLF4J is already included as a dependency of the parent project, I expect this line to be not necessary.